app/bundles/CoreBundle/Resources/views/Modal/keyboard_shortcuts.html.twig

```diff
@@ -16,78 +10,78 @@
-              'Del / ⌘+⌫' : 'mautic.keyboard.delete',
+            'e' : 'mautic.keyboard.edit',
+            'c' : 'mautic.keyboard.create',
+            '⌘+Enter' : 'mautic.keyboard.save',
+            'g+i' : 'mautic.keyboard.back',
+            'Del / ⌘+⌫' : 'mautic.keyboard.delete',
             } } %}
-          <h4 class="fw-sb">
-            {{ "mautic.keyboard.section.global" | trans }}
-          </h4>
-          {% for section, shortcut_list in shortcuts %}
+            <h4 class="fw-sb">
+              {{ "mautic.keyboard.section.global" | trans }}
+            </h4>
+            {% for section, shortcut_list in shortcuts %}
             <h5 class="mb-xs mt-md fw-sb">
               {{ ('mautic.keyboard.section.' ~ section)|trans }}
             </h5>
             <table class="shortcut-table bsv-xs">
               {% for key, description in shortcut_list %}
-                <tr class="mb-xs">
-                  <td class="shortcut-description">
-                    {{ description | trans }}
-                  </td>
-                  <td class="shortcut-keys text-right">
-                    {% set parts = key|split(' / ') %}
-                    {% for part in parts %}
-                        {% if not loop.first %}
-                            / <!-- Add the separator outside of the <kbd> tags -->
-                        {% endif %}
-                        {% set keys = part|split('+') %}
-                        {% for k in keys %}
-                            {% set title = '' %}
-                            {% if k == '⌫' %}
-                                {% set title = 'mautic.keyboard.backspace'|trans %}
-                            {% elseif k == '⌘' %}
-                                {% set title = 'mautic.keyboard.command'|trans %}
-                            {% endif %}
-                            <kbd{% if title %} title="{{ title }}" data-toggle="tooltip"{% endif %}>{{ k }}</kbd>
-                        {% endfor %}
+              <tr class="mb-xs">
+                <td class="shortcut-description">
+                  {{ description | trans }}
+                </td>
+                <td class="shortcut-keys text-right">
+                  {% set parts = key|split(' / ') %}
+                  {% for part in parts %}
+                  {% if not loop.first %}
+                  / <!-- Add the separator outside of the <kbd> tags -->
+                  {% endif %}
+                  {% set keys = part|split('+') %}
+                  {% for k in keys %}
+                  {% set title = '' %}
+                  {% if k == '⌫' %}
+                  {% set title = 'mautic.keyboard.backspace'|trans %}
+                  {% elseif k == '⌘' %}
+                  {% set title = 'mautic.keyboard.command'|trans %}
+                  {% endif %}
+                  <kbd{% if title %} title="{{ title }}" data-toggle="tooltip" {% endif %}>{{ k }}</kbd>
+                    {% endfor %}
```

Comment:
The CS, indentation is missing here.

Reply:
Fixed!

I always have this issue with the formatter in VS Code, do you know any extension that can properly format HTML with Twig?

Reply:
@andersonjeccel, I use PHPStrom, and it does help with indentation.

Google said this https://codeknight.co.uk/blog/getting-prettier-working-with-twig-craft-cms to be useful, 🤞